src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/MigrateRequest.kt

```diff
@@ -27,14 +28,23 @@ data class MigrateRequest(
   val restrictionType: NonAssociationRestrictionType,
 
   @Schema(
-    description = "Type of restriction, e.g. don't locate in the same cell",
+    description = "Explanation of why prisoners are non-associated",
     required = false,
     example = "John and Luke always end up fighting",
   )
   val comment: String? = null,
 
   @Schema(description = "Who authorised the non-association", required = false, example = "John Smith")
   val authorisedBy: String? = null,
+
+  @Schema(description = "Indicates that the NA is active", required = true, example = "false")
+  val active: Boolean,
```

Comment:
Could this be `isClosed` for consistency with the rest? I'm sure Syscon can easily map `active` to `isClosed` :)

Reply:
No, I want to keep the contract matching the data in NOMIS as these endpoint will be delete one live and migrated